client/src/pages/Rides.js

```diff
@@ -24,6 +29,18 @@ const PageWrapper = styled.div`
 
 const Rides = () => {
   const history = useHistory();
+  useEffect(() => {
+    const doFetch = async () => {
+      const today = getCurrentDate();
+      const todaysRides = await getDataByQuery({
+        collectionName: "tasks",
+        dataName: "date",
+        query: today,
+      });
+      return todaysRides;
```

Comment:
`todaysRides` is not used because `doFetch` doesn't expect a return value.
Maybe you like to save it in a state?

Reply:
Changed it to a state. Thanks for the advice. 🚀